inc/site-health.php

```diff
@@ -150,14 +150,15 @@ function output_compatibility_status_table( $plugins, $output = true, $incompati
 		<tbody>
 		<?php
 
-		// Filter out incompatible plugins. This allows us to re-use the status table for different types of compatibility issues.
-		if ( $incompatible && ( ! isset( $plugins['plugin_compatibility'] ) || $plugins['plugin_compatibility'] !== 'incompatible' ) ) {
-			$plugins = array_filter( $plugins, function ( $plugins ) {
-				return $plugins['plugin_compatibility'] === 'incompatible';
-			} );
-		}
+		foreach ( $plugins as $plugin => $field ) {
+			// Filter out incompatible plugins. This allows us to re-use the status table for different types of compatibility issues.
+			$plugin_compatibility = isset( $plugin['plugin_compatibility'] ) ? $plugin['plugin_compatibility'] : false;
```

Comment:
can we change this default value and change the logic or do we have a good reason to keep it false?